README.md

```diff
@@ -0,0 +1,357 @@
+based on the Routing Keys used to publish and bind queues. The ultimate goal of this plugin is to be used along the
+RabbitMQ MQTT plugin in order to use RabbitMQ as an MQTT broker with ACL-based authorization.
+
+Permissions for each routing key will be defined in Access Control Lists (ACLs) with a format similar to the one used
+for Mosquitto ACLs.
+
+## <a name="usage"/> Usage
+### <a name="deployment"/> Deployment
+
+In order to deploy the plugin:
+- Build the packages for distribution with `make dist`.
+- Move the generated `plugins/rabbitmq_topic_acl-<version>.ez`
+- Enable the plugins with the `rabbitmq-plugins` administration command.
+
+Take into account that once the plugin is enabled, it will enforce the authorization rules for all the RabbitMQ transactions.
+
+### <a name="administration"/> Administration
+
+Before starting to use the plugin, an ACL must be loaded in the database. For all the administrative actions, the
+RabbitMQ broker must be running with the plugin installed and enabled.
+
+There are two ways of managing the ACL DB contents:
+- Through the use of [ERL commands](#aclstore_commands) from the RabbitMQ Erlang console.
+- Through the use of the [AMQP Administration API](#amqp_administration).
+
+### <a name="examples"/> Examples of use
+
+Once the plugin is deployed and configured, and the ACL established, the plugin will start enforcing the authorizations.
+The `/docs` folder contains a series of examples of use that can be used to illustrate the behavior of the RabbitMQ broker
+with the activated plugin.
+
+Some examples make use of the [RabbitMQ ACL Tool](https://github.com/dmoranj/rabbit-acl-tool) commands for testing
+purposes. Check the documentation in the repository for further information.
+
+#### Example 1
+
+This example shows the basic behavior of the plugins for a very simple scenario: a news feed for a newspaper. The news
+feed has two kinds of topics:
+
+* `news/categories/<category>`: news channels for the different news categories offered by the newspaper. Journalists publish and
+read from this resource and the general audience subscribes to the messages.
+* `news/events`: private channel for the newspaper where the editor publishes events that must be covered by the journalists.
+ The editor publish to this resource and the journalists read from it.
+
+All the messages will be published to the `newsfeed` exchange.
+
+All the files for this example can be found in the `/docs/example1` folder.
+
+In order to setup the initial scenario, the RabbitMQ users must be created in the broker. Go to the `bin/` folder of
+your RabbitMQ instance and execute the `setupTest.sh` script. This script will create three users, on for each role in
+our example app: journalist, audience and editor. It also gives permissions for accessing all kind of entities in the
+default VHost. Take into account that this permissions are used at a different level than the routing key permissions
```

Comment:
Yes, it is. RabbitMQ supports multitenancy through that concept (similar to Apache's Virtual Hosts, for instance).